docs/Advanced Configuration.md

```diff
@@ -143,10 +143,15 @@ These settings are defined in `etc/erlinit.config`.
 +----------------------------+
 | p1*: Rootfs B (squashfs)   |
 +----------------------------+
-| p2: App Data  (FAT32)      |
+| p2: App Data  (ext4)       |
 +----------------------------+
 ```
 
+More information about how the App Data partition is initialized and mounted can be found
+in the documentation for `nerves_runtime` [Filesystem Initialization](https://hexdocs.pm/nerves_runtime/readme.html#filesystem-initialization)
+
+**Additional Partitions**
```

Comment:
I think this should be an h3 (prefix with `###`) instead of just bold text.
It would also be better to have this section be "Adding a Partition" to fit with the "Mounting the Partition" section that follows it.